packages/remix-dev/__tests__/build-test.ts

```diff
@@ -1,24 +1,102 @@
 import path from "path";
-import type { RollupOutput } from "rollup";
+import type * as esbuild from "esbuild";
 
 import { BuildMode, BuildTarget } from "../build";
-import type { BuildOptions } from "../compiler";
-import { build, generate } from "../compiler";
+import type { BuildOptions } from "../build";
+import { build } from "../compiler";
 import type { RemixConfig } from "../config";
 import { readConfig } from "../config";
 
-const remixRoot = path.resolve(__dirname, "../../../fixtures/gists-app");
+// Mock ESM-only modules as these are not well-supported in
+// the Jest environment available.
+jest.mock("xdm", () => ({
+  __esModule: true,
+  compile: () => ({}),
+}));
+jest.mock("remark-frontmatter", () => ({
+  __esModule: true,
+}));
+jest.mock("tsconfig-paths", () => ({
+  __esModule: true,
+  default: {
+    loadConfig() {
+      return {
+        resultType: "failed",
+      };
+    },
+  },
+}));
```

Comment:
This is not optimal but regardless of what I did to Babel or Jest these modules seemed to break Jest one way or another.